README.md

```diff
@@ -340,7 +340,7 @@ class SpockExample extends Specification {
         true
     }
 
-    @SnapshotName("should_use_extension_as_mehod_argument")
+    @SnapshotName("should_use_extension_as_method_argument")
```

Comment:
Can you also update the corresponding source code snippet

https://github.com/origin-energy/java-snapshot-testing/blob/b6996dfb4bd8a99ed2da0eac419a3a2d3c8dd8d5/java-snapshot-testing-spock/src/test/groovy/au/com/origin/snapshots/docs/SpockExample.groovy#L25